feat(guide/computed): add previous to computed

ferferga opened the pull request:
This was shipped silently in Vue 3.4. See [PR](https://github.com/vuejs/core/pull/9497)

NataliaTepluhina commented on src/guide/essentials/computed.md:
```diff
@@ -259,6 +259,114 @@ Now when you run `fullName.value = 'John Doe'`, the setter will be invoked and `
+    // When count is >=4, the last value that fulfilled our condition will be returned
+    // instead until count is less or equal to 3
+    alwaysSmall(previous) {
+      if (this.count >= 3) {
```
**question**: should this be `<=3` according to the explanation above?

NataliaTepluhina commented on src/guide/essentials/computed.md:
```diff
@@ -259,6 +259,114 @@ Now when you run `fullName.value = 'John Doe'`, the setter will be invoked and `
+// When count is >=4, the last value that fulfilled our condition will be returned
+// instead until count is less or equal to 3
+const alwaysSmall = computed((previous) => {
+  if (count.value >= 3) {
```
Same note about <=3

ferferga commented:
Good catch @NataliaTepluhina! Changed the logic a few times while working on this because I wasn't convinced in the examples that I was providing and completely missed this!

I also added the closing `</script>` tag that was missing from one of the Composition API's examples in my last commit.

NataliaTepluhina commented on src/guide/essentials/computed.md:
```diff
@@ -259,6 +259,115 @@ Now when you run `fullName.value = 'John Doe'`, the setter will be invoked and `
+  computed: {
+    alwaysSmall: {
+      get(previous) {
+        if (this.count >= 3) {
```
```suggestion
        if (this.count <= 3) {
```

NataliaTepluhina commented on src/guide/essentials/computed.md:
```diff
@@ -259,6 +259,115 @@ Now when you run `fullName.value = 'John Doe'`, the setter will be invoked and `
+
+const alwaysSmall = computed({
+  get(previous) {
+    if (count.value >= 3) {
```
```suggestion
    if (count.value <= 3) {
```

NataliaTepluhina commented:
Thank you @ferferga! I've updated also the examples for writable computeds and I think we're good to go now
